pkgs/applications/misc/styx/default.nix

```diff
@@ -15,27 +13,18 @@ stdenv.mkDerivation rec {
 
   server = "${caddy}/bin/caddy";
   linkcheck = "${linkchecker}/bin/linkchecker";
+  nixpkgs = pkgs.path;
```

Comment:
```suggestion
  nixpkgs = pkgs.path;
```

I'm finding this broad use odd. Maybe it is missing a wrapper for some specific libs?
Would be nice to check with someone with better knowledge...

@SuperSandro2000 Thoughts?

Reply:
it is used [here](https://github.com/styx-static/styx/blob/1d45ff744a34b09467dcd81adcab1737cb5dc357/src/styx.sh#L118).
I have no clue why it is called that way

Reply:
Please add a comment why this is done like it is.

Reply:
@ericsagnes or @blaggacao
could you chip in here, why we need a nixpkgs path here? Do you maybe have an alternative way of including styx in `nixpkgs` ?

Reply:
This can be considered technical debt. I have some impetus to chew through this upstream in the foreseeable future, as I pledged to modernize styx.

For the context (and timeframe) of this PR (thank you, btw), it should be considered inamovible.